inc/rulecollection.class.php

```diff
@@ -1130,8 +1130,7 @@ static function previewImportRules() {
 
                      //pass root entity and empty array (N/A value)
                      if (($action['field'] == "entities_id")
-                         && (($action['value'] == 0)
-                             || ($action['value'] == array()))) {
+                         && (empty($action['value']))) {
                         continue;
```

Comment:
To replace this part, we should be sure that $action['value'] cannot never contains false.

I don't like very much empty function who can be "expanded" as : 
empty($foo) : $foo == false

I had a lot of headaches to debug code with this function and false values return